euth/user_management/tests.py

```diff
@@ -1,6 +1,161 @@
+import pytest
+from django.core import mail
+from django.core.urlresolvers import reverse
+from django.contrib.auth.models import AnonymousUser, User, Permission
 from django.test import TestCase
 
-# Create your tests here.
-class DummyTestCase(TestCase):
-    def test_dummy(self):
-        self.assertTrue(True)
+from .forms import LoginForm
+from .models import Registration
+from .views import register_user
```

Comment:
It's easier to write and to read if you use `from . import views` instead of importing every view and model separately. 😄